packages/perseus/src/widgets/number-line/number-line.test.ts

```diff
@@ -4,7 +4,7 @@ import {testDependencies} from "../../../../../testing/test-dependencies";
 import * as Dependencies from "../../dependencies";
 import {renderQuestion} from "../__testutils__/renderQuestion";
 
-import {question1} from "./number-line.testdata";
+import {question1, snapshots} from "./number-line.testdata";
```

Comment:
This new import is just another question / PerseusItem, right? Why is it called `snapshots`? Why not `question3` (since `question2` is already defined)?

Also, why not use `question1` in the new tests? At a glance, I don't see much difference between `snapshots` and `question1`. If there is a significant difference, I'd prefer for it to be explicit in each test case where it matters (so, set explicitly when we create the `options` object).